### PR TITLE
Creating of Shared Device Multiple Users document

### PR DESCRIPTION
**Pull request by jpwalden**
Signed-off-by: jari-pekka walden <jari-pekka.walden@jolla.com>

### CODE REVIEW

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+* can make phone calls and send SMS/MMS if the admin has allowed this
+* can use cellular data for internet connections if the admin has allowed this
+* can use the apps installed by the admin
+* can create fingerprints for themselves
+* have security codes to protect the data of their own from other additional users
+* cannot change the settings of other users
+* do not have any Jolla accounts
+* do not have Jolla Store                         
+* do not have the Developer mode and Terminal available, cannot get super-user (root) rights
+* do not see Android App Support and Android app (not on OS release 3.4.0 at least)
```
Comment (jovirkku) [inline Support/Help_Articles/Shared_Device_Multiple_Users/README.md:91]: This part is not needed anymore: "(not on OS release 3.4.0 at least)"

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  </span>
+* <a href="TopMenu-Users.png"><img src="TopMenu-Users.png" alt="Topmenu users"></a>
+  <span class="md_figcaption">
+    Topmenu users
+  </span>
+</div>
+
+# What can different users do
+## The admin
+The admin has the same rights as the default user of Sailfish, "nemo", has had so far, up to OS release 3.3.0.
```
Comment (jovirkku) [inline Support/Help_Articles/Shared_Device_Multiple_Users/README.md:48]: _(this is originally my text - it looks outdated now)_  Let's change it  (line 48) to:
The admin equals the default user of Sailfish, so "defaultuser" (it used to be called "nemo" on Sailfish 3).

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+* Admin has a human-friendly name in "Settings > Users", with the role "Admin" shown underneath the name. The given name can be tens of characters long.
```
Comment (jovirkku) [inline Support/Help_Articles/Shared_Device_Multiple_Users/README.md:50]: _(again, my bad text)_  Let's go with this (line 50):
Admin can set a human-friendly name in "Settings > System > Users" to replace "defaultuser". His/her role "Admin" will be shown underneath the new name in the same menu. The given name can be tens of characters long.

**Review by jovirkku — CHANGES_REQUESTED** [review]

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md (hunk 1 above)
Comment (jpwalden) [reply Support/Help_Articles/Shared_Device_Multiple_Users/README.md]: Thanks, fixed

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md (hunk 2 above)
Comment (jpwalden) [reply Support/Help_Articles/Shared_Device_Multiple_Users/README.md]: Thanks, fixed

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md (hunk 3 above)
Comment (jpwalden) [reply Support/Help_Articles/Shared_Device_Multiple_Users/README.md]: Thanks, fixed

**Review by jovirkku — APPROVED** [review]
This is ok, imho.

File: Support/Help_Articles/Shared_Device_Multiple_Users/README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+---
+title: Shared Device Multiple Users
+permalink: Support/Help_Articles/Shared_Device_Multiple_Users/
+parent: Help Articles
+layout: default
+nav_order: 500
```
Comment (vigejolla) [inline Support/Help_Articles/Shared_Device_Multiple_Users/README.md:6]: This places the page before "SD Card Format and Encryption". In my alphabet D is before h.

Comment (jpwalden) [reply Support/Help_Articles/Shared_Device_Multiple_Users/README.md]: Fixed, that "SD Card Format and Encryption" wasn't there (or I missed it) when I pushed this so therefore I had wrong order, now that should be ok.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**